ShareX.ScreenCaptureLib/Shapes/ShapeManager.cs

```diff
@@ -93,9 +93,18 @@ public ShapeType CurrentTool
                     ClearTools();
                 }
 
-                if (previousTool != ShapeType.ToolSelect && currentTool != ShapeType.ToolSelect)
```

Comment:
You declared previousTool but it is not used anywhere?

Reply:
True, it became obsolete with the rework on this delect logic. I will remove it.